XM_Dinner/389_Torterra.yaml

```diff
@@ -0,0 +1,22 @@
+species: Torterra

```

Comment:
I'd cut on the bulk investment, it's weak offensively but not twice the bulk target weak